units/units.go

```diff
@@ -38,3 +38,13 @@ func FloatGweiToBigIntWei(gwei float64) *big.Int {
 
 	return wei
 }
+
+// FloatEthToBigIntWei returns the given eth as wei.
```

Comment:
Let's mention in the comment that precision might be lost, because it probably will be.